fs2-aws/src/test/scala/fs2/aws/kinesis/KinesisConsumerSpec.scala

```diff
@@ -62,6 +62,11 @@ class KinesisConsumerSpec
     commitableRecord.millisBehindLatest shouldBe recordsInput.build().millisBehindLatest()
   }
 
+  it should "complete if the scheduler shuts itself down (e.g. if it fails to initialise)" in new WorkerContext
+    with TestData {
+    stream.compile.drain.unsafeToFuture().futureValue
```

Comment:
This would have failed previously because the stream didn't terminate within two seconds (i.e. the patienceConfig value) even though `schedule.run()` returned after a second.